Add release notes 0.4.6 (#2332)
Napari/napari: docs/release/release_0_4_6.md

# napari 0.4.6

We're happy to announce the release of napari 0.4.6!
napari is a fast, interactive, multi-dimensional image viewer for Python.
It's designed for browsing, annotating, and analyzing large multi-dimensional
images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
rendering), and the scientific Python stack (numpy, scipy).

For more information, examples, and documentation, please visit our website:
https://github.com/napari/napari

## Highlights

> This release is the first that adds support for persistant settings in napari (#2212).

Reviewer: ```suggestion
This release is the first that adds support for persistent settings in napari (#2212).
```

Right now we just store the current theme and window geometry but we will be expanding 
this to include a full set of preferences in future releases.

We've also made plugin installation from our plugin dialog more flexible, including

> supporting instalation from a url, name, or file (#2319).

Reviewer: ```suggestion
supporting installation from anything supported by `pip`, such as a
package name, url, or local file (#2319).
```

## New Features
- Add the ability to show contours of labels (#2168)
- Add initial settings management for theme and window geometry (#2212)
- More flexible plugin install (from url, name or file)  (#2319)

## Improvements
- Use pydantic for viewer model (#2066)
- Use vectorized indexing for xarray arrays in labels (#2191)
- Add basic composite tree model (#2217)
- Add Qt Tree Model and View (#2222)
- Change status_format to return an existing string as-is (#2235)
- Add translator stub and example (#2254)
- Add hook implementation function name to plugin call order sorter (#2255)
- Add `theme` parameter to `get_stylesheet` (#2263)
- Add typing to event.py (#2269)
- Add `canvas.background_color_override` (#2270)
- QColoredSVGIcon class for dynamic colorized svg icons (#2279)
- Add EventedSet class (#2280)
- Refactor icon build process (#2285)
- Add more tooltips (#2292)

> - Make contour thickness setable (#2296)

Reviewer: ```suggestion
- Make contour thickness settable (#2296)
```
